requirements.txt

```diff
@@ -55,3 +55,4 @@ xlwt==1.3.*
 xlrd==1.1.0; python_version < '3.8'
 xlrd==1.2.0; python_version >= '3.8'
 pypiwin32 ; sys_platform == 'win32'
+pyopenssl==20.0.1
```

Comment:
No, I'll ask them. For the version is to be sure to have one version which work with the wrapper in urlib3 (it might be broken one day, so it's better to ensure the version we install)